iXBRLViewerPlugin/viewer/src/html/inspector.html

```diff
@@ -47,7 +47,7 @@ <h1 >Fact Properties</h1>
             </div>
           </div>
           <div class="search-entry">
-            <input id="ixbrl-search" type="text" ></input>
+            <input id="ixbrl-search" type="text" />
```

Comment:
I don't understand this commit.  These should be equally valid from an XHTML perspective?

Reply:
Yes.  The issue is that they get pre-processed by webpack's html-loader.  `<input></input>`  is technically invalid under (non-XML) HTML5, so html-loader minimises it into `<input>`, which is XML invalid.  `<input />` is both HTML and XHTML valid, and as we have `keepClosingSlash` turned on, html-loader preserves it.

Reply:
Ah, so "Fix HTML5 validity" might have been a better commit message.  Regardless, it all makes sense now, thanks.

Reply:
Well, the issue that it fixed was that if you created a viewer with a .xhtml extension, the viewer failed to load because it wasn't XML valid.  But more words were probably needed, as it took me 5 minutes to remind myself of the reason!